### PR TITLE
Update auth assertions readme

### PR DESCRIPTION
**Pull request by ryantmer**

### CODE REVIEW

File: packages/node_modules/brightspace-auth-assertions/README.md [hunk 1]
```diff
@@ -7,11 +7,12 @@
 ```js
 const
 	AssertionCompiler = require('brightspace-auth-assertions'),
-	AuthToken = require('brightspace-auth-token');
+	AuthToken = require('brightspace-auth-token'),
+	context = 'user'; // Valid contexts are 'user', 'tenant', and 'global'
```
Comment (omsmith) [inline packages/node_modules/brightspace-auth-assertions/README.md:11]:
```suggestion
	AuthToken = require('brightspace-auth-token');
```

File: packages/node_modules/brightspace-auth-assertions/README.md [hunk 2]
```diff
@@ -7,11 +7,12 @@
...
 
 const asserter = new AssertionCompiler()
 	.scope('valance', 'apps', 'manage').require()
-	.context(AssertionCompiler.contexts.User).require()
+	.context(context).require()
```
Comment (omsmith) [inline packages/node_modules/brightspace-auth-assertions/README.md:15]:
```suggestion
	.context('user').require() // Valid contexts are 'user', 'tenant', and 'global'
```

**Review by omsmith — APPROVED** [review]
Defining it top-level doesn't really match the intention of how you might consume it.

**Comment by ryantmer** [comment]
uh, yeah you right - not sure why I wrote it that way tbh 🤔 Fixed, and thank you! Finally can get this off my PR list after two years haha

**Pull request merged**